src/rgw/rgw_rados.cc

```diff
@@ -3176,6 +3176,12 @@ void RGWRados::finalize()
   delete gc;
   gc = NULL;
 
+  if (use_lc_thread) {
+    lc->stop_processor();
+  }
```

Comment:
`~RGWLC()` is already calling `stop_processor()`, so we shouldn't need this part